CONTRIBUTING.md

```diff
@@ -55,17 +55,16 @@ encouraged to try other DVC repositories -- especially real-world cases!
 
 - Go to the `demo/` directory of this repo.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Install [Yarn](https://yarnpkg.com/) if needed, and run `yarn install` to
+  setting up the project.
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Go back to the root of this repo (not `./demo`) and `yarn setup:venv` to setup
+  the Python virtual environment in the demo project.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+  > Python 3.8 is recommended, as some dependencies are not yet available for
+  > Python 3.9.
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- Return to `demo/`, activate the virtual environment, and run `dvc pull`.
```

Comment:
How do I activate the env? 😅 I'm not sure what kind of env it is even. There's no `activate` binary. I see:

```console
$ ls .env/bin 
__pycache__            dul-upload-pack        ftfy                   pip3                   python3.9              tqdm
cmark                  dulwich                futurize               pip3.10                shortuuid              wheel
convert-caffe2-to-onnx dvc                    jp.py                  pip3.9                 shtab
convert-onnx-to-caffe2 f2py                   normalizer             pygmentize             slugify
distro                 f2py3                  pasteurize             python                 tabulate
dul-receive-pack       f2py3.9                pip                    python3                torchrun
```

@iterative/vs-code 🙏🏼

Reply:
> There's no activate script.

I went through this again but running `yarn setup:env` as the first step, and there's an `activate` script now 🤷🏼 so this is answered I think. I'll update the instructions... ✔️

Reply:
Still not sure what kind of venv it is from the virtualenv-install.ts script, TBH. I guess readers don't need to know exactly but I'm curious: is it the same as a `python -m venv` ?

Reply:
It is `python -m venv` (which is similar to `virtualenv`). It a bit suboptimal btw, since `wheel` is not installed - some packages can be slower / unavailable with this setup.

Reply:
OK. Should we make an issue to deprecate virtualenv-install.ts or make it into platform-specific shell scripts? TBH I think just mentioning "create a `virtualenv`" with some links in the README should be good enough.